image/preseed/preseed.go

```diff
@@ -54,7 +53,10 @@ type CoreOptions struct {
 	// prepare image directory
 	PrepareImageDir string
 	// key to sign preseeded data with
-	PreseedSignKey string
+	PreseedSignKey *asserts.PrivateKey
```

Comment:
this change is strange because the code will not actually support a key that isn't manage by the KeypairManager built below, so I wouldn't do this change